CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -62,6 +62,19 @@ async def manual_report(state_file: UploadFile = File(None), cdc_file:  UploadFi
 
     return res
 
+# Runs the query on the state NBS ODSE database and outputs the results to a CSV file
+def automatic_report(year: int):
+    state_results = run_query(year)
+
+    folder_name = "temp"
+    if not os.path.exists(folder_name):
+        os.makedirs(folder_name)
+    result_save_to = f"./{folder_name}/state_results.csv"
```

Comment:
would make more sense for the file to be called state.csv instead of state_results.

Reply:
Also, the state csv file should be saved into a temp folder with the name made by uuid. Look at manual_report and the code `id = str(uuid.uuid4())
    os.makedirs(f"{folder_name}/{id}")`